Program.cs

```diff
@@ -34,7 +36,7 @@ private static void InitLog()
             logger.Info("Logger initialised");
         }
 
-        static Dictionary<string, Account> ReadFile(string filename)
+        static Dictionary<string, Account> ReadCSV(string filename)
```

Comment:
This file is getting quite long now. It might be worth splitting it into several files? 🙂